src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+import Tabs from './tabs/migrate-to-android.md'
+
+# Migrate to latest Android SDK
+
+If you have implemented the Android SDK 1.x versions, then this guide will help you understand the steps required to migrate your implementation to the latest Android SDK versions (2.0 or higher). In summary, you'll need to:
+
+1. [Switch Gradle dependencies from 1.x to 2.0](#switch-gradle-dependencies)
+2. [Update SDK initialization](#update-sdk-initialization)
+3. [Replace the deprecated API calls with the new API calls](#replace-the-deprecated-api-calls-with-the-new-api-calls1)
+
+## Switch Gradle dependencies
```

Comment:
`Switch Gradle dependencies to latest versions`
Also use the same naming in the linked steps above

Reply:
Can you also update the text for step 1 above to match this new title?